__test__/todo.test.js

```diff
@@ -0,0 +1,81 @@
+import indexModule from '../src/index.js';
+
+const { handleAddTask } = indexModule;
+const { dom } = indexModule;
+
+document.body.innerHTML = `<body>
```

Comment:
- [ ] Dear @Yemisirach, Good job importing your functions, But the better practice in such case would be to import/destructor from the main import. That is 

```javascript
import { handleAddTask, dom  } from '../src/index.js';
```